packages/@o3r/extractors/src/core/comparator/package-managers-extractors/npm-file-extractor.helper.ts

```diff
@@ -31,18 +31,22 @@ export async function getFilesFromRegistry(packageDescriptor: string, paths: str
   const tempDirPath = join(tmpdir(), tempDirName);
   let extractedFiles: { [key: string]: string } = {};
   mkdirSync(tempDirPath);
+  const [,packageName,packageRange] = sanitizeInput(packageDescriptor).match(/^(.*?)(?:\b@([^@]+))?$/) || [];
```

Comment:
not sure the `@` is forbidden in `prerelease` tag:

```suggestion
  const [,packageName,packageRange] = sanitizeInput(packageDescriptor).match(/^(.*?)(?:\b@(.+))?$/) || [];
```